### PR TITLE
Forming correct connection string

### PR DESCRIPTION
**Pull request by nyxathid**
New DB is running and already has data in it. You have to update your `config.json`. Ask @Henni for credentials. 

I could already get all artists from the DB. 

However, Storer has some BUGS!! 🐞 🐛 🐞 @LukasS96 #134

### CODE REVIEW

**Comment by nyxathid** [comment]
Some dummy data was already pushed to DB

**Comment by nyxathid** [comment]
ahh, I forgot to commit and push my latest changes
@LukasS96 could you take a look at this again?

File: src/storer.ts [hunk 1]
```diff
@@ -15,18 +15,38 @@ export class Storer {
...
+            }
             /*
+             @Lukas THIS DOESN'T WORK!
+
              Make sure that syncing to database is synchronous.
              Not that there is no {force: true} option here: We don't want to overwrite
              existing tables.
              */
-            context.sequelize.sync().then(() => {return this;});
+            context.sequelize.sync().then(() => {
```
Comment (Henni) [inline src/storer.ts]: `sync()` can be removed if your api submodule is up to date.

**Comment by LukasS96** [comment]
Is it necessary to call connect from storer and term-loader each? 
Connecting in processing-manager would erradicate the situation where a call to storer takes place when the db connection isn't established, therefore we wouldn't need those checks. See #134

**Comment by nyxathid** [comment]
@LukasS96 does #137 replace #135 ?
#137 looks better to me, should I close #135?

**Comment by LukasS96** [comment]
Yes :)

**Comment by nyxathid** [comment]
closed 👍

File: src/storer.ts [hunk 2]
```diff
@@ -15,18 +15,38 @@ export class Storer {
...
+    // will be set to true, when the connection is fine
+    private dbConnectionEstablished = false;
+
     constructor(){
+        // @Felix: you might want to change this -> pass everything as parameters to this function
+        let databaseURI = "postgresql://"
+            + CLI.parameters.dbUser + ":"
+            + CLI.parameters.dbPW + "@"
+            + CLI.parameters.dbHost + ":"
+            + CLI.parameters.dbPort + "/mcm";
```
Comment (kordianbruck) [inline src/storer.ts]: this should not be static. db can be differently named. not important for now.

**Pull request closed without merging**